backend/src/services/podcast/podcast.service.ts

```diff
@@ -250,6 +250,7 @@ class Podcast {
 
   public async delete(id: number): Promise<TPodcast> {
     const podcast = await this.#podcastRepository.getById(id);
+    const isOwner = true;
```

Comment:
```suggestion
    const isOwner = true;
```
not sure that is correct 🤔